src/views/debug/view_debug.hpp

```diff
@@ -9,7 +9,7 @@ namespace big::debug
 	extern void script_events();
 	extern void scripts();
 	extern void threads();
-	extern void animations();
+	extern void animations(std::string* dict = nullptr, std::string* anim = nullptr); // Can be used to retrieve animations
```

Comment:
Function params can be replaced by `std::string& dict, std::string& anim`.

An empty string can then be used to indicate the default state.
```cpp
if (!empty(dict) && !empty(anim))
{
    // code
}
```

Reply:
How about calling `animations` without any strings? For example the original call to render it in the debug window.